bindings/rust/wasmedge-sys/build.rs

```diff
@@ -1,13 +1,104 @@
-use std::path::PathBuf;
+use std::{env, path::PathBuf, process::Command};
 
 const WASMEDGE_H: &str = "wasmedge.h";
 
+fn build_wasmedge_project(hash: String) -> Paths {
+    let out_dir = env::var("OUT_DIR").expect("fail to get the out dir");
+
+    let wasmedge_src = format!("{}/wasmedge", &out_dir);
+    Command::new("git")
+        .args(&["init", &wasmedge_src])
+        .output()
+        .expect("fail to init wasmedge project");
+
+    Command::new("git")
+        .current_dir(&wasmedge_src)
+        .args(&[
+            "remote",
+            "add",
+            "origin",
+            "https://github.com/WasmEdge/WasmEdge.git",
+        ])
+        .output()
+        .expect("fail to add wasmedge upstream");
+
+    Command::new("git")
+        .current_dir(&wasmedge_src)
+        .args(&["fetch", "--tags", "origin", hash.trim()])
+        .output()
+        .expect("fail to fetch the commit");
+
+    Command::new("git")
+        .current_dir(&wasmedge_src)
+        .args(&["checkout", "FETCH_HEAD"])
+        .output()
+        .expect("fail to checkout the commit");
+
+    Command::new("cmake")
+        .current_dir(&wasmedge_src)
+        .args(&[
+            "-Bbuild",
+            "-GNinja",
+            "-DCMAKE_BUILD_TYPE=Release",
+            "-DWASMEDGE_BUILD_TESTS=ON",
+            "-DWASMEDGE_BUILD_AOT_RUNTIME=OFF",
```

Comment:
If you disabled the AoT runtime, you should also disable the aot related features in rust-sdk

Reply:
Thanks, AOT feature is added and enabled in default